packages/web/src/components/organisms/EntityDetails/EntityDetailsHeader.tsx

```diff
@@ -105,7 +106,7 @@ const EntityDetailsHeader: FC<EntityDetailsHeaderProps> = ({
           key="run-now-button"
           type="primary"
           onClick={onRun}
-          disabled={!details}
+          disabled={!details || isTestSuiteEmpty}
```

Comment:
Shouldn't we display a tooltip message why it's not possible to run it?

Reply:
added tooltip